src/main/java/frc/robot/subsystems/drivetrain/Drivetrain.java

```diff
@@ -238,6 +238,16 @@ public void resetOdometry(PathPlannerState state) {
         new Pose2d(state.poseMeters.getTranslation(), state.holonomicRotation));
   }
 
+  public void reassesPoseAfterDisabledVisionSubSystem() {
```

Comment:
How about naming the method resetPoseRotationToGyro? While this method will be invoked as a result of disabling the vision subsystem, it may be useful in other scenarios and the proposed name better reflects what it does.

Reply:
done